resources_pages/block_rep.md

```diff
@@ -1,40 +1,40 @@
 ---
 layout: page
-title: Block Representatives
+title: Student Representatives
 ---
 
-## MDS Block Representatives
+## MDS Student Representatives
 
-In the new academic year, there will be one set of block representatives in Term 1 (Blocks 1-3) and another set in Term 2 (Blocks 4-6). Block representatives volunteer to represent their peers, acting as a liaison between students and MDS leadership. This allows us to monitor how each block is progressing and make adjustments if needed.
+In the new academic year, there will be one set of student representatives in Term 1 (Blocks 1-3) and another set in Term 2 (Blocks 4-6). Student representatives volunteer to represent their peers, acting as a liaison between students and MDS leadership. This allows us to monitor how the program is progressing and make adjustments if needed.
 
-### Block Representative Roles
+### Student Representative Roles
 
-In addition to the general role of gathering and communicating feedback, block representatives will also have specific roles, such as focusing on academics, career development, social activities, health and well-being, etc. The exact roles will be determined when the program begins. Please note that these guidelines apply to the MDS Vancouver program. If you are enrolled in MDS Computational Linguistics (MDS CL), the policies might differ.
+In addition to the general role of gathering and communicating feedback, student representatives will also have specific roles, such as focusing on academics, career development, social activities, health and well-being, etc. The exact roles will be determined when the program begins. Please note that these guidelines apply to the MDS Vancouver program. If you are enrolled in MDS Computational Linguistics (MDS CL), the policies might differ.
```

Comment:
Should we explicitly list these roles? There will be five representatives (per section) for academic, social, sports and recreation, career and professional development, and mental health and wellness.

Reply:
I consider we should list these exact five roles (omit the "_etc._"): _academic_, _social_, _sports and recreation_, _career and professional development_, and _mental health and wellness_. We don't have to define them right now (we could do that after our meeting).